src/host/cookedRead.cpp

```diff
@@ -55,9 +55,9 @@ bool CookedRead::HasHistory() const noexcept
     return _pCommandHistory != nullptr;
 }
 
-CommandHistory& CookedRead::History() noexcept
+std::shared_ptr<CommandHistory> CookedRead::History() noexcept

```

Comment:
I would prefer if this handed out a reference to the command history like it used to, it shouldn't be allowing other classes to share in the history ownership from here.

Reply:
OK.